app/views/dashboard/_host_reports_status_links.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= host_reports_searchable_links _('Host Reports with changes'), "report_changes > 0", :change_hosts, format%>
+<%= host_reports_searchable_links _('Host Reports without changes'), "report_nochanges > 0", :nochange_hosts, format %>
+<%= host_reports_searchable_links _('Host Reports with failures'), "report_failures > 0" , :failure_hosts, format %>
```

Comment:
Found a bug = the widget always shows summary for particular format (Ansible, Puppet) but the query does not reflect it. These queries must be "report_changes > 0 and report_format = 'Ansible'" or something like that. 

However, to be able to do that, you need to update host_extensions.rb:

```
      scoped_search relation: :host_reports, on: :change, rename: :report_changes, only_explicit: true
      scoped_search relation: :host_reports, on: :nochange, rename: :report_nochanges, only_explicit: true
      scoped_search relation: :host_reports, on: :failure, rename: :report_failures, only_explicit: true
```

Here add three more lines to add such a scoped search.